### DISCUSSION TITLE
Exploring Keystatic - a blog article

### DISCUSSION THREAD

**Discussion by armno** [Thinkmill/keystatic#839] [Show and tell]
Hi 👋,

I wrote a blog article about Keystatic, specifically adding Keystatic to my existing Astro blog 🚀 . 

I noted down what I have learned along the way. Here it is: https://armno.in.th/blog/exploring-keystatic/

It's my first attempt trying out Keystatic and I like it a lot. This blog article covers:
- Why I find Keystatic useful to manage Markdown content
- Adding Keystatic to my existing Astro blog - what are the steps and issues I found, and how I work around them.
- (bonus section) What I like about Keystatic and what I think could be better.

I hope you find the article useful. Any feedback would be really appreciated 🙏.

![content-focused](https://github.com/Thinkmill/keystatic/assets/911894/fe5b90ee-f813-4768-98c7-f5ed46d28a45)

Cheers.
Armno.

**Comment by simonswiss** [comment]
Whoaaa, great amount of details in this post — goof work! 🎉

**Reply by armno** [reply]
@simonswiss Thank you!
